### PR TITLE
adds trained deblender that performs reasonably well

### LINKED ISSUE #26: retrain deblender
use dataset in #19 

- [x] consider system to have multiple models that can be tagged and then evaluated , just make sure to log what is happening with date
- [ ]  retrain with different training seeds deblender encoder (and use step size in #23 , but perhaps we will need to try a bunch)

### LINKED ISSUE #32: look at some examples of trained deblender performance on training images
- [ ] consider running on toy example with true centroids
- [ ] how close are the example residual images to noise? 

Could consider ideas from #56

### PR DESCRIPTION
**Pull request by ismael-mendoza**
- Closes #32
- Closes #26 

Performance looks pretty good on average on images from prior (training dataset), hard to say exactly how much since mostly dim galaxies. 

Looking at toy examples for that part of the paper, the reconstruction is a lot better, never greater than 5 absolute residual across separations (using the true centroid) -- should check again with detection encoder trained later. 

Finally, there are some examples of shredding when attempting to reconstruct a very dim source close to a very bright source (see below). Unclear if this will happen in reality as we wouldn't detect the dim source?

### CODE REVIEW

**Comment by ismael-mendoza** [comment]
shredding examples (images from prior): 

Screenshot 2024-09-27 at 09 06 01
Screenshot 2024-09-27 at 09 10 13
Screenshot 2024-09-27 at 09 06 35
Screenshot 2024-09-27 at 09 07 27
Screenshot 2024-09-27 at 09 09 01

**Comment by ismael-mendoza** [comment]
example different separations: 

Screenshot 2024-09-27 at 12 20 00
Screenshot 2024-09-27 at 12 21 00
Screenshot 2024-09-27 at 12 20 13
Screenshot 2024-09-27 at 12 20 25
